### PR TITLE
Remove AI service from OpenOfficePanel

### PR DESCRIPTION
**Pull request by InAnYan**
Just a more AI dependency. But it's not that much, because it's still needed for groups..

### Mandatory checks

~- [ ] Change in `CHANGELOG.md` described in a way that is understandable for the average user (if applicable)~
~- [ ] Tests created for changes (if applicable)~
- [x] Manually tested changed features in running JabRef (always required)
~- [ ] Screenshots added in PR description (for UI changes)~
- [x] [Checked developer's documentation](https://devdocs.jabref.org/): Is the information available and up to date? If not, I outlined it in this pull request.
- [x] [Checked documentation](https://docs.jabref.org/): Is the information available and up to date? If not, I created an issue at <https://github.com/JabRef/user-documentation/issues> or, even better, I submitted a pull request to the documentation repository.

### CODE REVIEW

**Review by subhramit — APPROVED** [review]

File: src/main/java/org/jabref/gui/sidepane/SidePaneContentFactory.java [hunk 1]
```diff
@@ -66,7 +66,6 @@ public Node create(SidePaneType sidePaneType) {
             case OPEN_OFFICE -> new OpenOfficePanel(
                     tabContainer,
                     preferences,
-                    aiService,
```
Comment (subhramit) [inline src/main/java/org/jabref/gui/sidepane/SidePaneContentFactory.java:69]: Remove the import, checkstyle happy

**Review by subhramit — CHANGES_REQUESTED** [review]

File: src/main/java/org/jabref/gui/sidepane/SidePaneContentFactory.java (hunk 1 above)
Comment (subhramit) [reply src/main/java/org/jabref/gui/sidepane/SidePaneContentFactory.java:69]: Nvm, was just one line so i removed it

Comment (subhramit) [reply src/main/java/org/jabref/gui/sidepane/SidePaneContentFactory.java:69]: Nope, seems like that was wrong. Reverting.

Comment (subhramit) [reply src/main/java/org/jabref/gui/sidepane/SidePaneContentFactory.java:69]: Yep, should be fine now

**Review by subhramit — APPROVED** [review]

File: src/main/java/org/jabref/gui/sidepane/SidePaneContentFactory.java (hunk 1 above)
Comment (InAnYan) [reply src/main/java/org/jabref/gui/sidepane/SidePaneContentFactory.java:69]: @subhramit thanks!

**Pull request merged**